executor/executor_test.go

```diff
@@ -3371,7 +3371,7 @@ func (s *testSuite3) TestCurrentTimestampValueSelection(c *C) {
 	tk.MustQuery("select id from t where t0 = ?", t0).Check(testkit.Rows("1"))
 	tk.MustQuery("select id from t where t1 = ?", t1).Check(testkit.Rows("1"))
 	tk.MustQuery("select id from t where t2 = ?", t2).Check(testkit.Rows("1"))
-	time.Sleep(time.Second / 2)
```

Comment:
There are two calls of `now()` around this `sleep()`.
One is used to insert a record `t0=now()` and the other is used to update it `set t0=now()`.
Before this PR, since `now()` rounds the result, so we can assume that these two calls of `now()` must get two time with different seconds.
After this PR, this assumption is broken, so I increase the waiting time to ensure they can get two time with different seconds.